src/Fieldset/README.md

```diff
@@ -0,0 +1,29 @@
+# Fieldset
+
+Wraps many inputs in a border and allows validating them as one form control.
+
+## API
+
+### `dangerIconDescription` (string or element; optional)
+`dangerIconDescription` can be used to provide a screen-reader description of the "exclamation" icon used in the validation message displayed when `isValid` is false and `themes` includes "danger". The default is an empty string.
+
+### `children` (string, element, or arrays of strings or elements; optional)
+`children` specifies the elements that the Fieldset will wrap. The default is an empty string.
```

Comment:
Is there a reason to explicitly point out the `children` prop vs. the implicit use of the `children` prop when using `Fieldset` as a wrapper.

I could see how somebody might get confused with this `prop` and do something like

```javascript
<FieldSet children={<noblah />)>
  <blah />
</FieldSet>
```